frontend/package.json

```diff
@@ -133,19 +133,19 @@
   },
   "dependencies": {
     "@patternfly-5/patternfly": "npm:@patternfly/patternfly@5.4.2",
-    "@patternfly/patternfly": "^6.0.0",
+    "@patternfly/patternfly": "^6.2.0-prerelease.2",
     "@patternfly/quickstarts": "^6.2.0-prerelease.4",
     "@patternfly/react-catalog-view-extension": "^6.1.0-prerelease.3",
-    "@patternfly/react-charts": "^8.1.0",
-    "@patternfly/react-component-groups": "^6.1.0",
+    "@patternfly/react-charts": "^8.2.0-prerelease.13",
+    "@patternfly/react-component-groups": "^6.2.0-prerelease.4",
     "@patternfly/react-console": "^6.0.0",
-    "@patternfly/react-core": "^6.1.1-prerelease.1",
-    "@patternfly/react-icons": "^6.1.0",
+    "@patternfly/react-core": "^6.2.0-prerelease.15",
+    "@patternfly/react-icons": "^6.2.0-prerelease.2",
     "@patternfly/react-log-viewer": "^6.1.0",
-    "@patternfly/react-styles": "^6.1.0",
-    "@patternfly/react-table": "^6.1.0",
-    "@patternfly/react-templates": "^6.1.0",
-    "@patternfly/react-tokens": "^6.1.0",
+    "@patternfly/react-styles": "^6.2.0-prerelease.2",
+    "@patternfly/react-table": "^6.2.0-prerelease.16",
+    "@patternfly/react-templates": "^6.2.0-prerelease.16",
+    "@patternfly/react-tokens": "^6.2.0-prerelease.2",
     "@patternfly/react-topology": "^6.1.0",
```

Comment:
Do you think we could wait for a prerelease for this package to pick up this fix? https://github.com/patternfly/react-topology/pull/271

Reply:
We can.  The intention is for us to land on final versions.  So there will probably be multiple PRs with version bumps.